.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk:
 env:
   matrix:
     - GRADLE_VERSION=default
-    - GRADLE_VERSION=5.2.1
+    - GRADLE_VERSION=5.3.1
```

Comment:
To keep test with 5.2, please do not replace existing lines, instead add a new line for `5.3.1`

Reply:
got it, done! 👍